### PR TITLE
fix: remove assertion operators (!!)

### PR DESCRIPTION
**Pull request by lekaha**
issues: https://github.com/lekaha/playone-android/issues/10

### CODE REVIEW

**Comment by lekaha** [comment]
I have reformed the code with our style by the way

File: mobile-ui/src/main/java/com/playone/mobile/ui/view/recycler/RecyclerViewAdapter.kt [hunk 1]
```diff
@@ -10,19 +10,24 @@ import java.util.*
...
+    private val factoryMap: Map<Int, ViewHolderFactory>,
+    private val binderMap: Map<Int, ViewHolderBinder>
+) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
 
     private val modelItems = ArrayList<DisplayableItem<*>>()
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
-        factoryMap[viewType]!!.createViewHolder(parent)
+        factoryMap[viewType]?.createViewHolder(parent)
+        ?: throw IllegalStateException("The view factory of the type is null. " +
```
Comment (pokk) [inline mobile-ui/src/main/java/com/playone/mobile/ui/view/recycler/RecyclerViewAdapter.kt:23]: The msg `"The view factory of the type is null. viewType = $viewType"` will be better.

File: mobile-ui/src/main/java/com/playone/mobile/ui/view/recycler/RecyclerViewAdapter.kt [hunk 2]
```diff
@@ -10,19 +10,24 @@ import java.util.*
...
+                                       "viewType = $viewType")
 
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
         val item = modelItems[position]
-        binderMap[item.type()]!!.bind(holder, item)
+        binderMap[item.type()]?.bind(holder, item)
+        ?: throw IllegalStateException("The view binder of the type is null. " +
```
Comment (pokk) [inline mobile-ui/src/main/java/com/playone/mobile/ui/view/recycler/RecyclerViewAdapter.kt:29]: The msg `"The view binder of the type is null. viewType = ${item.type()}"` will be better.

File: mobile-ui/src/main/java/com/playone/mobile/ui/view/recycler/RecyclerViewAdapter.kt (hunk 1 above)
Comment (lekaha) [reply mobile-ui/src/main/java/com/playone/mobile/ui/view/recycler/RecyclerViewAdapter.kt]: You mean ` ${viewType}`, right?

Comment (lekaha) [reply mobile-ui/src/main/java/com/playone/mobile/ui/view/recycler/RecyclerViewAdapter.kt:23]: What do you mean?

**Pull request merged**